packages/portal/components/Stage.js

```diff
@@ -0,0 +1,57 @@
+import { Stage as PixiStage } from '@inlet/react-pixi'
+import { makeStyles } from '@material-ui/core/styles'
+import React, { useEffect } from 'react'
+import { useMeasure } from 'react-use'
+import Controls from './stage/Controls'
+
+const useStyles = makeStyles(
+  () => ({
+    root: {
+      position: 'relative',
+      width: '100%',
+      height: '100%',
+      minHeight: 300,
+      minWidth: 200,
+      overflow: 'hidden',
+    },
+    stage: {
+      position: 'absolute',
+      overflow: 'hidden',
+    },
+  }),
+  { name: 'Stage' },
+)
+
+const stageOptions = {
+  backgroundColor: 0x1099bb,
+  resolution: window.devicePixelRatio,
+}
+
+let app
```

Comment:
Shouldn't we use `useRef` for this?

Reply:
`@inlet/react-pixi` has a `useApp` hook, which can be used in child components of the stage, I guess... didn't work in the initial component.